scripts/Synapse/analyzeLinkageAlgorithms.ipynb

```diff
@@ -978,8 +997,21 @@
         "from phdi.linkage import DIBBS_ENHANCED\n",
         "\n",
         "new_algo = copy.deepcopy(DIBBS_ENHANCED)\n",
-        "new_algo[0][\"kwargs\"][\"threshold\"] = 0.85\n",
-        "new_algo[1][\"kwargs\"][\"threshold\"] = 0.85\n",
+        "col_thresholds = {\n",
```

Comment:
Are you thinking that the thresholds will be set up by users in the config file or calculated automatically? Just wondering where this will live once we finish testing

Reply:
Tbh I don't actually think k-fold validation code will be needed at all in the SDK, so my plan was once we acquire the thresholds from our testing, we just store them as part of the DIBBs algos that we define as constants in the SDK. If the user wants to change any of the thresholds, we can just use the same API that they would use to change the overall threshold.